provider/lib/cronAlarm.js

```diff
@@ -123,4 +124,24 @@ module.exports = function(logger, newTrigger) {
         return limit;
     }
 
+    function isStrict(strict) {
+        /**
+         * If the strict variable is not passed from alarmWebAction(User doesn't define strict value),
+         * then the ALARM_DELAY_DEFAULT_STRICT environment variable value is used.
+         */
+        if(strict === undefined || strict === null) {
+            return delayDefaultStrict;
+        }
+
+        /**
+         * "true"(string)   -> true
+         * "false"(string)  -> false
+         * "True"(string)   -> true
+         * "False"(string)  -> false
+         * true(boolean)    -> true
+         * false(boolean)   -> false
+         */
+        return String(strict).toLowerCase() === "true";
```

Comment:
@KeonHee 
If this is to let users use both `"true"` and `true` for the parameter, it would be great to leave comments about it.

Reply:
Updated user documents as your comment :)

Reply:
oh ok, I meant the comments on codes but never mind.